tests/integration/targets/inventory_aws_ec2/templates/inventory_with_hostnames.yml.j2

```diff
@@ -0,0 +1,21 @@
+plugin: amazon.aws.aws_ec2
+aws_access_key_id: '{{ aws_access_key }}'
+aws_secret_access_key: '{{ aws_secret_key }}'
+{% if security_token | default(false) %}
+aws_security_token: '{{ security_token }}'
+{% endif %}
+regions:
+- '{{ aws_region }}'
+keyed_groups:
+- prefix: tag
+  key: tags
+hostnames:
+# can also be specified using
+# - tag:Tag1,Tag2
+# or
+# - tag:Tag1=Test1,Tag2=Test2
```

Comment:
If possible I'd like to see an integration test of at least one of these.
"tag:Tag1,Tag2" wouldn't be the 'usual' way to write tags in a filter (see #582), so if that leaked into a filter it's more likely to break things than "tag:Tag1,tag:Tag2" appearing in a filter.

Reply:
Added the integration tests for `- tag:Tag1=Test1,Tag2=Test2`.